src/Workload/Microsoft.Maui.Controls.Sdk/Sdk/Sdk.targets

```diff
@@ -32,6 +36,11 @@
   </ItemGroup>
 
   <Import Project="../targets/BundledVersions.targets" />
-  <Import Project="Microsoft.Maui.Controls.Sdk.targets" Condition=" '$(UseMaui)' == 'true' " />
+  <!--
+  these targets currently bring in a mixture of controls logic (xamlc, default xaml items etc) and generic logic (singleproject)
+  they should really be split up and conditioned separately so projects can get singleproject logic without all the controls stuff.
+  perhaps singleproject should be a UseSingleProject that works by itself?
+  -->
```

Comment:
Do we want to do this now?

Reply:
I don't think we need to do this in this PR. This PR is more about fixing default item groups.

@mhutch is this ready to come out of "draft"?

Reply:
Yeah. I mostly put this in draft because I hadn't talked about it with anyone or tested it.